gluten-core/src/main/scala/io/glutenproject/utils/QueryPlanSelector.scala

```diff
@@ -59,11 +57,11 @@ abstract class QueryPlanSelector[T <: QueryPlan[_]] extends Logging {
           s"plan:\n${plan.treeString}\n" +
           "=========================")
     }
-    val conf: Option[String] = session.conf.getOption(GlutenConfig.GLUTEN_ENABLE_KEY)
-    val ret = conf.flatMap((x: String) =>
-      Try(x.toBoolean).toOption).getOrElse(GlutenConfig.GLUTEN_ENABLE_BY_DEFAULT)
-    logInfo(s"shouldUseGluten: $ret")
-    ret & validate(plan)
+    val glutenEnabled = session.conf.
+      get(GlutenConfig.GLUTEN_ENABLE_KEY, GlutenConfig.GLUTEN_ENABLE_BY_DEFAULT.toString).toBoolean
+    logInfo(s"shouldUseGluten: $glutenEnabled")
```

Comment:
It's really noisy, can we suppress this logging ? I think debug level is enough...

Reply:
Yes, it looks too verbose. I will do a fix. Thanks!